include/seqan3/argument_parser/detail/format_help.hpp

```diff
@@ -418,6 +415,15 @@ class format_help : public format_help_base<format_help>
             std::cout << '\n';
     }
 
+    /*!\brief Format string as in_bold.
```

Comment:
Shouldn't it be "Format string in bold"?

Reply:
yes

Reply:
```suggestion
    /*!\brief Format string in bold.
```